docker-compose.yml

```diff
@@ -2,13 +2,15 @@ version: '2.4'
 
 services:
   nextcloud-ephemeral-master:
+    environment:
+      - PORT
     build:
       context: .
       dockerfile: php7.4.Dockerfile
       args:
         BRANCH: master
     ports:
-      - "80:80"
+      - "${PORT}:80"
```

Comment:
This needs a default (in the form of a `.env` file, for example), otherwise it will resolve to a blank string when `PORT` is not in the env.

Reply:
Added .env